doc/connectivity/bluetooth/api/audio/shell/bap.rst

```diff
@@ -114,7 +114,11 @@ Or using connect command:
    uart:~$ bt connect <address>
    uart:~$ gatt exchange-mtu
    uart:~$ bap discover sink
-   uart:~$ bap connect sink 0
+   uart:~$ bap config sink 0
+   uart:~$ bap qos
+   uart:~$ bap enable
+   uart:~$ bap connect
+   uart:~$ bap start
```

Comment:
Trying this against a Samsung earbud, the audio flows fine, but it already seems to start streaming after calling 'connect'

![image](https://github.com/zephyrproject-rtos/zephyr/assets/1014678/2009ab47-cdfd-4ed2-b124-644082456d2d)

Reply:
Your comment is for the `Connect and establish a source stream` when you are doing it on a sink stream. See the above steps for sink streams

Reply:
yes... I see that now (was following the steps in the previous push that had config sink... all good... and approved :D